pinboard/pinboard.py

```diff
@@ -172,9 +172,12 @@ def __call__(self, *args, **kwargs):
                 401: exceptions.PinboardAuthenticationError,
                 403: exceptions.PinboardForbiddenError,
                 500: exceptions.PinboardServerError,
+                503: exceptions.PinboardServiceUnavailable,
             }
-            Error = error_mappings[e.code]
-            raise Error(e.url, e.code, e.msg, e.hdrs, e.fp)
+            if e.code in error_mappings:
+                Error = error_mappings[e.code]
+                raise Error(e.url, e.code, e.msg, e.hdrs, e.fp)
+            raise
```

Comment:
Would prefer an `else` here and a more explanatory exception (i.e., perhaps explain that the status code isn't accounted for?). Otherwise looks good!